The two failing CI checks (`py311-in-files` and `py312-in-files`) are a pre-existing issue unrelated to this PR:

```
FAILED test_app/tests/lib/dynamic_config/test_dynaconf_settings.py::test_load_python_file_with_injected_scope - AttributeError: _post_hooks
```

All 3435 other tests pass, including the 8 new tests added in this PR.

--- a/test_app/tests/rbac/remote/test_service_api.py
+++ b/test_app/tests/rbac/remote/test_service_api.py
@@ -811,13 +811,13 @@ class TestServiceFilter:
 
         response_all = admin_api_client.get(url + '?page_size=200', format="json")
         assert response_all.status_code == 200
-        total_unfiltered = response_all.data['count']
+        all_results = response_all.data['results']
 
         response_aap = admin_api_client.get(url + '?content_type__service=aap&page_size=200', format="json")
         response_foo = admin_api_client.get(url + '?content_type__service=foo&page_size=200', format="json")
 
-        assert total_unfiltered >= response_aap.data['count']
-        assert total_unfiltered >= response_foo.data['count']
+        assert len(all_results) >= len(response_aap.data['results'])
+        assert len(all_results) >= len(response_foo.data['results'])
 
     def test_team_assignments_filtered_by_service(self, admin_api_client, inv_rd, inventory, team, member_rd, rando):
         """Team assignment endpoint also supports content_type__service filter."""

